testsuite/tests/singlecluster/authorino/authorization/opa/external_registry/conftest.py

```diff
@@ -24,5 +24,5 @@ def authorization(authorization, opa_policy_expectation):
     Adds OPA policy. Rego query is located on external registry (Mockserver).
     Policy accepts requests that contain `header`.
     """
-    authorization.authorization.add_external_opa_policy("opa", opa_policy_expectation, 1)
+    authorization.rules.authorization.add_external_opa_policy("opa", opa_policy_expectation, 1)
```

Comment:
I would make it backwards compatible  and make `rules` default if nothing is specified. Will result in much smaller PR without any significant sideeffect